docs/api-reference/next/image.md

```diff
@@ -14,11 +14,12 @@ description: Enable Image Optimization with the built-in Image component.
 <details>
   <summary><b>Version History</b></summary>
 
-| Version   | Changes                  |
-| --------- | ------------------------ |
-| `v10.0.5` | `loader` prop added.     |
-| `v10.0.1` | `layout` prop added.     |
-| `v10.0.0` | `next/image` introduced. |
+| Version   | Changes                                          |
+| --------- | ------------------------------------------------ |
+| `v11.0.0` | Static image support and blur placeholder added. |
```

Comment:
```suggestion
| `v11.0.0` | Static image imports and `placeholder` prop added. |
```

Reply:
Also we added the 'blurDataURL' prop. I chose the initial wording because specifically mentioning static and both props was kind of throwing off the formatting.

Reply:
Oh yeah almost forgot. We should mention all of those:

```
Added static imports for `src`. Added `placeholder` and  `blurDataURL` props.
```

Reply:
Done